web/integration-test/src/test/java/org/apache/gravitino/integration/test/web/ui/utils/BaseWebIT.java

```diff
@@ -82,12 +82,10 @@ protected void clickAndWait(final Object locator) throws InterruptedException {
       wait.until(ExpectedConditions.elementToBeClickable(locatorElement(locator)));
 
       locatorElement(locator).click();
-      Thread.sleep(ACTION_SLEEP_MILLIS);
```

Comment:
The issue lies in here and L90. You should not delete these two lines unless you have alternatives.